src/Roslyn.Diagnostics.Analyzers/CSharp/CSharpApplyTraitToClass.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Roslyn.Diagnostics.Analyzers;
+
+namespace Roslyn.Diagnostics.CSharp.Analyzers
+{
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = nameof(CSharpApplyTraitToClass))]
+    [Shared]
+    public sealed class CSharpApplyTraitToClass : AbstractApplyTraitToClass<AttributeSyntax>
```

Comment:
nit: Consider renaming to end in "Refactoring"

Reply:
➡️ Not planning to change this for now. For better or worse, the current name matches the majority pattern seen in this project.